openupgrade_scripts/apriori.py

```diff
@@ -76,6 +90,20 @@
     # OCA/survey
     "survey_description": "survey",
     # OCA/...
+    
+    # Tvtmaaddons
```

Comment:
@hieu1211 sửa như này nhé chú
```suggestion
    # Viindoo/tvtmaaddons
```